src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline

```diff
@@ -810,6 +810,7 @@ module Library
       # @example
       #   require "library/v1/library_service_api"
       #
+      #   Alignment = Google::Example::Library::V1::SomeMessage2::SomeMessage3::Alignment
```

Comment:
@Landrito @swcloud Not a comment on this PR per se, but this could be fragile if multiple types with the same simple name (e.g. `Alignment`) are needed, correct?